apps/assisted_tagging/schema.py

```diff
@@ -20,9 +21,11 @@
     MissingPredictionReview,
     WrongPredictionReview,
 )
+from lead.models import Lead
 from .enums import (
     DraftEntryPredictionStatusEnum,
     AssistedTaggingPredictionDataTypeEnum,
+    AutoEntryExtractionTypeEnum
```

Comment:
Adding a trailing comma will make the next diff clean.
```suggestion
    AutoEntryExtractionTypeEnum,
```